README.md

```diff
@@ -69,7 +56,7 @@ To get a local copy up and running, follow these steps.
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/228936141-3232012b-f41e-4a4a-9a05-b449c1533b2e.png)

**[OPTIONAL]**

- [ ] It would be great if you make your description more detailed to make your readme file more professional and descriptive 👍 and let the viewer know that what this app provides 😊